client/my-sites/site-settings/delete-site/index.jsx

```diff
@@ -299,7 +299,7 @@ class DeleteSite extends Component {
 								<p>
 									{ translate(
 										"To delete this site, you'll need to contact our support team. Deletion can not be undone, " +
```

Comment:
Hi! I've found a possible matching string that has already been translated **18** times:
`translate( 'To delete this site, you'll need to contact our support team. Deletion can not be undone, and will remove all content, contributors, domains, and upgrades from this site.' )` *ES Score: 12*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).